python/ray/state.py

```diff
@@ -377,6 +377,54 @@ def profile_table(self):
 
         return dict(result)
 
+    # SANG-TODO Add functions.
+    def placement_group_table(self, placement_group_id=None):
+        self._check_connected()
+
+        if placement_group_id is not None:
+            placement_group_id = ray.PlacementGroupID(
+                hex_to_binary(placement_group_id.hex()))
+            placement_group_info = (
+                self.global_state_accessor.get_placement_group_info(
+                    placement_group_id))
+            if placement_group_info is None:
+                return {}
+            else:
+                placement_group_info = (gcs_utils.PlacementGroupTableData.
+                                        FromString(placement_group_info))
+                return self._gen_placement_group_info(placement_group_info)
+        else:
+            raise NotImplementedError(
+                "Get all placement group is not implemented yet.")
+
+    def _gen_placement_group_info(self, placement_group_info):
+        def get_state(state):
+            if state == 0:
```

Comment:
How about define `PlacementGroupState` in gcs_utils.py? just like `ray.gcs_utils.ActorTableData.ALIVE`.

Reply:
Sounds good. Didn't know we could do that.